### ISSUE TITLE
No THEAD when stacked="md"

### ISSUE THREAD

**Issue by warlord0** [bootstrap-vue/bootstrap-vue#3932]
### Describe the bug

Following an upgrade to rc28 our table headers have disappeared.

![b-table-fail](https://user-images.githubusercontent.com/9975039/63577314-04e64480-c586-11e9-834b-c1ac327dd2f4.png)

### Steps to reproduce the bug

Most of our usage uses `stacked="md"` If I remove the attribute the table headers then show as expected.

```html
    <b-table
      id="contractsTable"
      ref="table"
      :items="$_tableProvider"
      :fields="fields"
      :current-page="currentPage"
      :per-page="perPage"
      :filter="filter"
      :api-url="apiUrl"
      stacked="md"
      show-empty
      striped
      no-local-sorting
      no-local-filtering
    >
```

### Expected behavior

What it looks like without the `stacked` attribute.

![b-table-ok](https://user-images.githubusercontent.com/9975039/63577291-fc8e0980-c585-11e9-936c-92de5f53e0cd.png)

### Versions

**Libraries:**

- BootstrapVue: 2.0.0-rc28
- Bootstrap: 4.3.1
- Vue: 2.6.10

**Environment:**

- Device: PC
- OS: Debian Linux (buster)
- Browser: Chrome
- Version: 76.0.3809.100 (Official Build) (64-bit)

**Comment by tmorehouse** [comment]
Duplicate: https://github.com/bootstrap-vue/bootstrap-vue/issues/3886
Fixed via: https://github.com/bootstrap-vue/bootstrap-vue/pull/3887

Will be available in the next release which will be 2.0.0 stable (hopefully next week)

**Comment by tmorehouse** [comment]
BootstrapVue v2.0.0 stable has been released. See https://bootstrap-vue.js.org/docs/misc/changelog
